roles/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 name = "stratum_v2_roles"
-version = "0.0.0"
+version = "1.0.0"
 authors = ["The Stratum v2 Developers"]
 edition = "2021"
 rust-version = "1.65.0"
```

Comment:
Same here: shouldn't be 1.75.0? @Fi3

Reply:
ack https://github.com/stratum-mining/stratum/pull/785/commits/5ac08c2a20802c787a914b1cad4613f7748de1c4